openPrompt/described_structure/readme.md

```diff
@@ -0,0 +1,30 @@
+## described_structure
+### Description
+With a described structure there is even less information given, consider the following prompt, now with a described JSON structure
+
+### Examples
+
+#### Prompt1
```

Comment:
shall we use markdown list grammar for multiple prompts?

Reply:
It seems inappropriate to use list grammar both Prompt and Response, I need some examples